tests/drivers/i2c/i2c_target_api/testcase.yaml

```diff
@@ -32,3 +32,10 @@ tests:
       - nucleo_f091rc
     extra_configs:
       - CONFIG_APP_DUAL_ROLE_I2C=y
+  drivers.i2c.target_api.single_role:
+     platform_allow:
+       - frdm_mcxn947/mcxn947/cpu0
+       - mimxrt1170_evk@B/mimxrt1176/cm7
```

Comment:
I do not have revB but only RevA, which works fine in single mode. so please add mimxrt1170_evk/mimxrt1176/cm7, and its board overlay as well